profiles/permissions.py

```diff
@@ -20,3 +23,35 @@ def has_object_permission(self, request, view, profile):
             return True
 
         return profile.user == request.user
+
+
+class CanSeeIfNotPrivate(BasePermission):
+    """
+    Only owner of a profile can view and edit their profile.
+    Verified micromaster users can view other profiles to limited view only if
+    account_privacy of profile is set to public public_to_mm
+    """
+
+    def has_permission(self, request, view):
+        "Implementation of the permission class"
```

Comment:
All docstrings should have three quotes on each side: https://www.python.org/dev/peps/pep-0008/#documentation-strings